python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -130,19 +129,28 @@ def __init__(self, *args, **kwargs):
                                                                      "NO_DATA", "DETECTED_NEGATIVE"]))
 
     @pipeBase.timeMethod
-    def run(self, templateExposure, exposure, subtractedExposure, psfMatchingKernel):
+    def run(self, exposure, templateExposure, subtractedExposure, psfMatchingKernel,
+            xcen=None, ycen=None, var1=None, var2=None):
         """! Perform decorrelation of an image difference exposure.
```

Comment:
I see the reference. Thanks. Maybe I_1 and I_2 should be I_s and I_t in DMTN-021, to make it explicit?